config/projects-to-test/openjdk25-excluded.files

```diff
@@ -70,7 +70,7 @@
   <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]SuperInit[\\/]SuperInitGood.java$"/>
 </module>
 
-<!-- until https://github.com/checkstyle/checkstyle/issues/17810 -->
+<!-- until https://github.com/checkstyle/checkstyle/issues/18462 -->
```

Comment:
Please see related issue (https://github.com/checkstyle/checkstyle/issues/18462) for reason why I haven't removed these `BeforeExecutionExclusionFileFilter`s.

In short, 10 checks throw null-pointer exceptions when they see a compact source file. This breaks CI job `.ci/no-exception-test.sh openjdk25-with-checks-nonjavadoc-error`. 

If checkstyle had a `checks` property on `BeforeExecutionExclusionFileFilter`, we could have narrowed down these filters to only the 10 checks. Unfortunately, that isn't possible.